src/cluster.bootstrap/Akka.Management.Cluster.Bootstrap/ClusterBootstrapSettings.cs

```diff
@@ -151,8 +151,7 @@ public static ContactPointSettings Create(Config config)
                 var contactPointConfig = config.GetConfig("akka.management.cluster.bootstrap.contact-point");
                 var fallback = contactPointConfig.GetString("fallback-port");
                 var fallbackPort = string.IsNullOrWhiteSpace(fallback) || fallback == "<fallback-port>"
-                    ? config.GetInt("akka.management.http.port")
```

Comment:
Do not depend on HOCON config values of another plugin. The HOCON config is not the single source of truth for configuration anymore, since the final settings value can be influenced by a `Setup` class.